fMRIVolume/scripts/DistortionCorrectionAndEPIToT1wReg_FLIRTBBRAndFreeSurferBBRbased.sh

```diff
@@ -430,10 +445,53 @@ case $DistortionCorrection in
                 --smoothingfwhm="2" \
                 --inputdir="$WD"
         fi
-            
+
 
         ;;
 
+    ${NONE_METHOD_OPT})
+
+            # NOTE: To work with later code a number of fake files are created 
+            #       with names that do not reflect the true nature of the file
+            #       this is to be considered an initial fix for datasets that 
+            #       do not include data that enable distortion correction and
+            #       will be replaced with a cleaner code. 
+
+
+            log_Msg "---> No distortion correction"
+
+            ScoutExtension=""
```

Comment:
Weren't you intending to set `ScoutExtension` in this case to something like `orig` or `original`?  That also seems safer (to make sure that we aren't overwriting a file that we don't intend to overwrite).

Reply:
I had it original at first. Then it seemed strange to be named `Scout_gdc_orig`. But I take your point. I'll add orig back in.

Reply:
How about we call it "_nosdc"?

Reply:
Ok. `_nosdc` it is.